pisp_app/lib/Screens/lookup_payee.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:logger/logger.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/log_printer.dart';
+
+import 'transaction_amount.dart';
+
+class LookupPayee extends StatefulWidget {
```

Comment:
I don't see this being a `StatefulWidget` (?)
Perhaps if we still want to use this, we could convert it into a `StatelessWidget`?

Reply:
Makes sense. Changed.